docs/integrations/prometheus.txt

```diff
@@ -122,7 +125,7 @@ This example startup script includes the default setup docker commands provided
 
 #. Launch Prometheus with the provided `prometheus.yml
    <https://github.com/determined-ai/works-with-determined#observability-tools>`__ configuration
-   file.
+   file. This file needs a manual change to replace the placeholder master address.
```

Comment:
```suggestion
   file. **This file needs a manual change to replace the placeholder master address.**
```

Reply:
No! We don't use typography for syntax highlighting. Do not use `**`.
Everything in the doc is equally important or it wouldn't be there. Users need to read the doc, not scan for highlighted syntax. Similarly, we don't use `.. note::`